src/devices/Bmxx80/Bmxx80Base.cs

```diff
@@ -53,61 +53,54 @@ protected Bmxx80Base(byte deviceId, I2cDevice i2cDevice)
             }
         }
 
-        /// <summary>
-        /// Sets the pressure sampling.
-        /// </summary>
-        /// <param name="sampling">The <see cref="Sampling"/> to set.</param>
-        public void SetPressureSampling(Sampling sampling)
-        {
-            byte status = Read8BitsFromRegister(_controlRegister);
-            status = (byte)(status & 0b1110_0011);
-            status = (byte)(status | (byte)sampling << 2);
-            _i2cDevice.Write(new[] { _controlRegister, status });
-        }
+        private Sampling _pressureSampling;
```

Comment:
NIT: very dumb thing, but would you mind moving these backup fields to the top of the class? That is usually how we have most of the bindings so just in order to be consistent.